priv/ecto_repo/migrations/00000000000002_add_timestamps.exs

```diff
@@ -0,0 +1,32 @@
+defmodule FunWithFlags.Dev.EctoRepo.Migrations.EnsureColumnsAreNotNull do
+  use Ecto.Migration
+  #
+  # Use this migration to add the ecto `inserted_at` and `updated_at` columns
+  # to the table created using the `CreateFeatureFlagsTable` migration
+  # from versions `<= 1.10.0`.
+  #
+  # https://hexdocs.pm/ecto_sql/Ecto.Migration.html#timestamps/1
+  #
+  # If the table has been created with a migration from `>= 1.11.0`,
```

Comment:
I am just guessing that this change would necessitate a minor version bump. Note that this PR **does not** actually change any version numbers.